CODE_OF_CONDUCT.md

```diff
@@ -1,36 +1,114 @@
 # Contributor Covenant Code of Conduct
 
 ## Our Pledge
+Diversity is one of the greatest strengths a community can have and many times
+that strength is born from the friction that can only come through sharing of
+differing perspectives.
 
-In the interest of fostering an open and welcoming environment, we as
+In the interest of fostering an open, welcoming, and encouraging environment, we as
```

Comment:
I'm fond of encouraging as well - and for the record, Habitat has been one of the most encouraging, welcoming and open communities I've ever participated in.